src/java/org/apache/cassandra/tools/nodetool/stats/StatsTableComparator.java

```diff
@@ -66,7 +66,10 @@ public class StatsTableComparator implements Comparator<StatsTable>
                                                        "space_used_by_snapshots_total", "space_used_live",
                                                        "space_used_total", "sstable_compression_ratio", "sstable_count",
                                                        "table_name", "write_latency", "writes", "max_sstable_size",
-                                                       "local_read_write_ratio", "twcs_max_duration"};
+                                                       "local_read_write_ratio", "twcs_max_duration", "sai_local_query_latency",
```

Comment:
"local_read_latency_ms", "local_write_latency_ms" are with the "ms" postfix, but here we use it without any postfix. Any particular reason for this? It feels like the naming should be preserved.